CONTRIBUTION.md

```diff
@@ -0,0 +1,76 @@
+
+# How to contribute
+
+Pull requests should include the following:
+- link to an issue that describing the problem, or a description in the pull request describing the issue and what it solves
+- unit tests (if applicable)
+- storybook (if applicable)
+- a separate commit that contains the ouput from running `npm run docs`
+
+## Setup guide
+
+### Cloning repository
+
+Start by cloning the repository to desired directory.
+
+```
+git clone https://github.com/equinor/esv-intersection.git
+```
+
+### Install dependencies
+
+Install all dev dependencies defined in package.json using node.
+
+```
+npm install
+```
+
+## Usage
+
+### Storybook
+
+```
+npm run storybook
+```
+
+### Document generation
+
+```
+npm run docs
+```
+
+Three part process:
+
+1. Deletes the docs-folder, if it exists.
```

Comment:
If the folder is actually called docs, I'd write 
```
Delete the `docs` folder
```

Reply:
you can use ```suggestion at the start of the formatting, while selecting the line you want to change. this way the suggestion can be commited directly from here

Reply:
```suggestion
1. Deletes the `docs` folder, if it exists.
```